### PR TITLE
docs(main-2.1): add rebalance doc

### PR DESCRIPTION
**Pull request by Gala-R**

### CODE REVIEW

File: docs/zh_CN/tasks/configure-emqx-rebalance.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+- **提升系统扩展性**：由于 MQTT 连接是基于 TCP/IP 协议的长连接，当集群扩容后，旧节点的连接不会自动迁移到新节点上。如果希望新节点承载旧节点上部分负载，可以通过重平衡功能，将旧节点上的负载平滑地迁移到新节点上，从而使整个集群负载更加均衡，提高系统的吞吐量，响应速度以及资源利用率，使系统更好地扩展。
+
+- **降低运维成本**：如果系统中某些节点负载过高或过低，需要对这些节点进行手动调整，而通过重平衡，可以自动调整节点的负载，降低运维成本。
+
+关于 EMQX 集群负载重平衡可以参考文档：[重平衡](https://docs.emqx.com/zh/enterprise/v4.4/advanced/rebalancing.html#%E9%87%8D%E5%B9%B3%E8%A1%A1)。
+
+:::tip
+
+集群负载重平衡功能仅在 EMQX 企业版 4.4.12 版本才开放。
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/configure-emqx-rebalance.md:21]:
```suggestion
集群负载重平衡要求 EMQX 企业版必须大于等于 4.4.12
```

File: docs/en_US/tasks/configure-emqx-rebalance.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+The value of rebalancing mainly has the following two points:
+
+- **Improve system scalability**: Due to the persistent nature of MQTT connections, connections to the original nodes will not automatically migrate to the new nodes when the cluster scales. To address this, you can use the load rebalancing feature to smoothly transfer connections from overloaded nodes to newly-added ones. This process ensures a more balanced distribution of load across the entire cluster and enhances throughput, response speed, and resource utilization rate.
+- **Reduce O&M costs**: For clusters with unevenly distributed loads, where some nodes are overloaded while others remain idle, you can use the load rebalancing feature to automatically adjust the load within the cluster. This helps achieve a more balanced distribution of work and reduces operation and maintenance costs.
+
+For EMQX cluster load rebalancing, please refer to the document: [Rebalancing](https://docs.emqx.com/en/enterprise/v4.4/advanced/rebalancing.html#rebalancing).
+
+:::tip
+
+The cluster load rebalancing function is only available since EMQX Enterprise 4.4.12.
```
Comment (Rory-Z) [inline docs/en_US/tasks/configure-emqx-rebalance.md:20]:
```suggestion
The cluster load rebalancing requirements EMQX Enterprise Edition must be greater than or equal to 4.4.12
```

**Review by Rory-Z — COMMENTED** [review]
Double check the main branch PR

**Review by Rory-Z — COMMENTED** [review]
Double check the main branch PR

File: docs/zh_CN/tasks/configure-emqx-rebalance.md (hunk 1 above)
Comment (Gala-R) [reply docs/zh_CN/tasks/configure-emqx-rebalance.md]: > 

done

File: docs/en_US/tasks/configure-emqx-rebalance.md (hunk 2 above)
Comment (Gala-R) [reply docs/en_US/tasks/configure-emqx-rebalance.md]: > 

done

**Review by Rory-Z — APPROVED** [review]

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**